tags/interceptors.go

```diff
@@ -22,6 +31,14 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 	}
 }
 
+// StreamClientInterceptor returns a new streaming server interceptor for OpenTracing.
```

Comment:
This just injects in a ctx tags for opentracing, opentracing is not the only usecase.